R/texreg.R

```diff
@@ -1925,6 +1925,467 @@ print.texregTable <- function(x, ...) {
   cat(x, ...)
 }
 
+#' Create coefficient plots from statistical model output using \pkg{ggplot2}.
+#'
+#' Create coefficient plots of \R regression output using \pkg{ggplot2}.
+#'
+#'  The \code{plotreg} function produces coefficient plots (i.e., forest plots 
```

Comment:
Please make sure there is only one space after each `#'` in this description block.